string name
--------------------------------
bool inImage
string personInImage
float32 x
float32 y
float32 z
float32 tetha
float32 phi
float32 sci
